app/components/Edit/EditScheduleDay.js

```diff
@@ -21,7 +21,7 @@ class EditScheduleDay extends Component {
           <div>
             <input type="time" defaultValue={this.props.getDayHours(day, "opens_at", index)} data-key={day} data-field="opens_at" onChange={(e) => this.props.handleScheduleChange(day, index, "opens_at", e.target.value)} />
             <input type="time" defaultValue={this.props.getDayHours(day, "closes_at", index)} data-key={day} data-field="closes_at" onChange={(e) => this.props.handleScheduleChange(day, index, "closes_at", e.target.value)} />
-            {index > 0 ? <button onClick={() => this.props.removeTime(day, index)}className="remove-time">x</button> : ''}
+            {index > 0 ? <button onClick={() => this.props.removeTime(day, index)}className="remove-time icon-button"><i className="material-icons">&#xE872;</i></button> : ''}
```

Comment:
I am assuming that `&#xE872;` is ascii for `x`, but only from the change you made. Is it possible to save that as a descriptive variable so that no one has to guess in the future?

Reply:
fixed to proper material-icons usage